Top-Level Requirements.md

```diff
@@ -70,10 +72,12 @@
 * 8.8 Upon completing the mining procedure, the mining subsystem **shall**:
     * 8.8.1 Define an “obstacle” representing the digging hole and send it to the navigation subsystem.
     * 8.8.2 Return the mining arm to the starting position.
+* 8.9 The mining subsystem **shall** expose a mechanism to reset arm postition.
 ### 9.  Dumping Subsystem
-* 9.1 As a precondition the rover **shall** be facing east, relatively close to the dumping receptacle.
-* 9.2 The dumping subsystem **shall** perform the localization procedure prior to the start of the  dumping procedure.
-* 9.3 The dumping procedure **shall**:
-    * 9.3.1 Orient itself and turn to face east.
-    * 9.3.2 Back up to within a set tolerance of the wall, maintaining eastern orientation.
-    * 9.3.3 Raise the mining bin and dump the regolith at a set speed.
+* 9.1 As a precondition the rover **shall** be facing west, relatively close to the dumping receptacle.
```

Comment:
Is it East or West?

Reply:
East I think we start east and end navigation facing east

Reply:
Well, whatever it is it should be consistent with the definition at the very top.